R/file_increment.R

```diff
@@ -16,16 +17,17 @@
 #' @returns Invisibly returns a vector of logical values specifying
 #' whether or not the file was successfully renamed.
 #' @seealso [jitter()]
-file_increment <- function(i, verbose = FALSE,
+file_increment <- function(path, i, verbose = FALSE,
                            pattern = "^[CcPRw][a-zA-Z]+\\.sso|summary\\.sso|\\.par$") {
   if (verbose) {
     message("Renaming output files to have names like Report", i, ".sso")
   }
+
   ignore <- file.copy(
-    from = dir(pattern = pattern),
+    from = dir(path = path, pattern = pattern, full.names = TRUE),
     to = gsub("par", "par_", gsub(
       "\\.sso|(\\.par)",
-      paste0("\\1", i, ".sso"), dir(pattern = pattern)
+      paste0("\\1", 0, ".sso"), dir(path = path, pattern = pattern, full.names = TRUE)
```

Comment:
Since the call to `file_increment()` in `jitter()` uses `i = 0` it shouldn't be hardwired here. However, we could also just get rid of the function entirely since `jitter()` is the only place it gets used. I think the role it is playing in this parallelized version is just to backup the original files but that shouldn't be needed since the function automatically creates new separate directories.

Reply:
Ah yes, thanks for catching this. I changed it just to test locally but forgot to change it back.